GAE/src/org/waterforpeople/mapping/app/web/rest/ActionRestService.java

```diff
@@ -372,7 +373,12 @@ private String copyProject(Long targetId, Long folderId) {
             Survey surveyCopy = SurveyUtils.copySurvey(survey, surveyDto);
             surveyCopy.setSurveyGroupId(savedProjectCopy.getKey().getId());
             survey.setAncestorIds(surveysAncestorIds);
-            surveyDao.save(surveyCopy);
+            long copyId = surveyDao.save(surveyCopy).getKey().getId();
+            if (survey.getKey().getId() == projectCopy.getNewLocaleSurveyId()) {
+                //original was the registration survey for its survey group
+                projectCopy.setNewLocaleSurveyId(copyId); //fix it
+                surveyGroupDao.save(projectCopy);
```

Comment:
Perhaps we should used the `saveProjectCopy` instead of the `projectCopy` variable?